ipynb/primary-processing/1-get_sequences_and_metadata.md

```diff
@@ -29,17 +29,10 @@ Now let's actually fetch the study data. `fetch_study` will only pull down acces
 Now that we have the sequences and sample information, let's merge all the data into a single file to ease downstream processing.
 
 ```python
->>> form_path = lambda acc, ext: agenv.get_existing_path(os.path.join('1', '%s.%s' % (acc, ext)))
-...
->>> bash_compatible_sequence_files = ' '.join([form_path(acc, 'fna') for acc in accessions])
->>> !cat $bash_compatible_sequence_files > $agp_sequences
-...
->>> if len(accessions) > 1:
-...     qiime_compatible_mapping_files = ','.join([form_path(acc, 'txt') for acc in accessions])
-...     !merge_mapping_files.py -m $qiime_compatible_mapping_files -o $agp_metadata
->>> else:
-...     src = form_path(accessions[0], 'txt')
-...     !cp $src $agp_metadata
+>>> sample_sequence_files = ' '.join(agenv.get_files(chp_path, suffix='fna'))
+>>> !cat $sample_sequence_files > $agp_sequences
```

Comment:
it's likely that this won't work when there are thousands of files due to the bash command line length limit